inspirehep/search/api.py

```diff
@@ -98,14 +98,17 @@ class LiteratureSearch(InspireSearch):
     class Meta:
         index = "records-hep"
         doc_types = "hep"
-        default_filter = DefaultFilter(Q("match", _collections="Literature"))
+        default_filter = DefaultFilter(Q())
 
     def query_from_iq(self, query_string):
         """Initialize ES DSL object using INSPIRE query parser.
         :param query_string: Query string as a user would input in INSPIRE's search box.
         :type query_string: string
         :returns: Elasticsearch DSL search class
         """
+        if not is_superuser_or_cataloger_logged_in():
+            user_query = Q(IQ(query_string, self) & Q("match", _collections="Literature"))
```

Comment:
it's better to use filters for this kind of stuff, they are faster as they don't influence the score and can be cached (see [here](https://www.elastic.co/guide/en/elasticsearch/reference/5.6/query-filter-context.html) for more details).